webapp/pages/settings/my-social-media.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <ds-card :header="$t('settings.social-media.name')">
+    <ds-space
+      v-if="socialMediaLinks"
+      margin-top="base"
+      margin="x-small"
+    >
+      <ds-list>
+        <ds-list-item
+          v-for="link in socialMediaLinks"
+          :key="link.url"
+        >
+          <a :href="link.url">
+            <img
+              :src="link.favicon"
+              alt="Social Media link"
+              width="16"
+              height="16"
+            >
+            {{ link.url }}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-28T17:44:04Z" title="Thursday, March 28th 2019, 6:44:04 pm +01:00">Mar 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,86 @@
+<template>
+  <ds-card :header="$t('settings.social-media.name')">
+    <ds-space
+      v-if="socialMediaLinks"
+      margin-top="base"
+      margin="x-small"
+    >
+      <ds-list>
+        <ds-list-item
+          v-for="link in socialMediaLinks"
+          :key="link.url"
+        >
+          <a :href="link.url">
+            <img
+              :src="link.favicon"
+              alt=""
+            >
+            {{ link.url }}
```


should this be the entire url @roschaefer, we were thinking of either just the username, or just having the icon as the link
![Screenshot at 2019-03-28 14:43:48](https://user-images.githubusercontent.com/26943915/55180228-ec553800-5167-11e9-838f-3bb75ce53c6a.png)

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-29T08:31:52Z" title="Friday, March 29th 2019, 9:31:52 am +01:00">Mar 29, 2019</time>_

---


I think it has some advantage to see the url, people like to know where the icon points to.

If you know how to get the username out of an arbitrary url, go for it :thinking: :woman_shrugging:

Reply:
> [<img alt="Lulalaby" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Lulalaby) **Authored by [Lulalaby](https://github.com/Lulalaby)**
_<time datetime="2019-03-29T09:34:22Z" title="Friday, March 29th 2019, 10:34:22 am +01:00">Mar 29, 2019</time>_

---


I prefer that only the username is filtered from the URL and displayed with the icon.
Otherwise it might get too long.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-29T13:30:42Z" title="Friday, March 29th 2019, 2:30:42 pm +01:00">Mar 29, 2019</time>_

---


@mattwr18 if it turns out that filtering the username from the URL is not that easy, don't spend too much time on it. I agree with @Lulalaby but a quick fix could also be that you simply shorten the URL like `/some/path...`.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-29T17:32:11Z" title="Friday, March 29th 2019, 6:32:11 pm +01:00">Mar 29, 2019</time>_

---


I think potential it could be not too complicated @roschaefer... the regex that @Kachulio1 wrote :point_up: separates the the beginning of the URL from the username already, so potentially we just need to save what it cuts as the username, but we'll have a look at it

Reply:
> [<img alt="Lulalaby" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Lulalaby) **Authored by [Lulalaby](https://github.com/Lulalaby)**
_<time datetime="2019-03-29T20:09:01Z" title="Friday, March 29th 2019, 9:09:01 pm +01:00">Mar 29, 2019</time>_

---


yeah, just do it! 😸